lib/prelude.dx

```diff
@@ -2236,6 +2236,9 @@ def reflect {n} [Ix n] (i:n) : n =
 def reverse {n a} (x:n=>a) : n=>a =
   for i. x.(reflect i)
 
+def wrap (n:Type) [Ix n] (i:Nat) : n = 
```

Comment:
I feel like `wrap` is a very general name to use in the prelude, so I'd rather either (1) put this function in the example, (2) rename to `wrap_periodic` to emphasize what it does.

Reply:
Yes that sounds better, renamed it to `wrap_periodic`  now.